dbt/adapters/redshift/impl.py

```diff
@@ -115,6 +116,34 @@ def valid_incremental_strategies(self):
     def timestamp_add_sql(self, add_to: str, number: int = 1, interval: str = "hour") -> str:
         return f"{add_to} + interval '{number} {interval}'"
 
+    def _get_cursor(self):
+        return self.connections.get_thread_connection().handle.cursor()
+
+    def get_columns_in_relation(self, relation):
```

Comment:
I think you need to throw the `@available` decorator on this method for it to be available in the jinja context. I would be surprised if the tests passed since you use it in your new test.

Reply:
Tests passed and I was able to call this new overriden definition of get_columns_in_relation() in my test project macro.  I saw that other adapters didn't have the decorator on top like [this one](https://github.com/dbt-labs/dbt-bigquery/blob/2bce97721fb7f7733e0a49ec97430072720d7033/dbt/adapters/bigquery/impl.py#L307)...not 100% sure but I'm guessing maybe its because dbt-core's definition of [get_columns_in_relation](https://github.com/dbt-labs/dbt-core/blob/fe9c875d32554cf36a08a94a6ee5d356066d80f8/core/dbt/adapters/base/impl.py#L576) already had decorator declared?

Reply:
Oh interesting, TIL...